perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -90,11 +91,17 @@
                      'The size of the working set, in GB. If not given, use '
                      'the full size of the device.',
                      lower_bound=0)
+flags.DEFINE_integer('run_for_minutes', 10,
+                     'Repeat the job scenario(s) for the given number of '
+                     'minutes. Only valid when using --generate_scenarios. '
+                     'When using multiple scenarios, each one is run for the '
```

Comment:
Could you document here the interaction between `--run_for_minutes` and `MINUTES_PER_JOB`?